server/test-data/compile.sh

```diff
@@ -240,7 +240,7 @@ function CompileLinux {
 		RemoveTarget $target
 		mkdir -p $(dirname $target)
 
-		$LINUX_GCC $flags -fomit-frame-pointer -fno-strict-aliasing -fno-exceptions $src -c -o /tmp/$name-$archname.o
+		$LINUX_GCC $flags -fPIC -fomit-frame-pointer -fno-strict-aliasing -fno-exceptions $src -c -o /tmp/$name-$archname.o
```

Comment:
A point to make for new builds on linux, that the mix&match isn't ok between non-pic versions and pic versions.